ctapipe/io/tests/test_merge.py

```diff
@@ -32,10 +32,10 @@ def compare_stats_table(in1, in2, merged, table):
     t2 = read_table(in2, table) if table in in2 else None
 
     if t1 is None:
-        raise ValueError(f" table {table} not present in {in1.filename}")
```

Comment:
Shouldn't `read_table` throw an error like this already if the table is not in the file? Then this code is not needed and neither is the `if table in file else None` above

Reply:
From the debugging point of view, it would be better to land in the failing test, right? Not in the `read_table` method?